apps/sync/js/crypto/fxsyncwebcrypto.js

```diff
@@ -42,7 +42,7 @@ const FxSyncWebCrypto = Object.freeze((() => {
     // The number 64 here comes from
     // (256 bits for AES + 256 bits for HMAC) / (8 bits per byte).
     return KeyDerivation.hkdf(kBByteArray,
-                              StringConversion.rawStringToByteArray(
+                              StringConversion.asciiStringToByteArray(
```

Comment:
Please rename `asciiStringToByteArray` to `stringToAsciiByteArray`, so its relationship with `stringToUtf8ByteArray` is clearer. Also please add a comment about why we're not using `stringToUtf8ByteArray` here.